components/admin_console/admin_definition.jsx

```diff
@@ -2454,6 +2454,21 @@ const AdminDefinition = {
                             )
                         ),
                     },
+                    {
+                        type: Constants.SettingsTypes.TYPE_TEXT,
+                        key: 'LdapSettings.AdminFilter',
+                        label: t('admin.ldap.adminFilterTitle'),
+                        label_default: 'Admin Filter:',
+                        help_text: t('admin.ldap.adminFilterFilterDesc'),
+                        help_text_default: '(Optional) Enter an AD/LDAP filter to use when searching for admin objects. Only the users selected by the query will be able to access Mattermost as Admins.\n \nNote: If this filter is removed/changed, active admins will not be demoted to a member and will retain their Admin role. Admins can be demoted in **System Console > User Management**.\n \n \nExisting members that are identified by this attribute as a admin will be demoted from a admin to a member when they are asked to login next. The next login is based upon Session lengths set in **System Console > Session Lengths**. It is highly recommend to manually demote admins to members in **System Console > User Management ** to ensure access is restricted immediately.',
```

Comment:
"Note: If this filter is removed/changed, active admins will not be demoted to a member and will retain their Admin role." 
Based on prior review this isn't true. They are being promoted and demoted on next login. 
@thefactremains - Can you take a look at this text and help?

Reply:
Oops! My error. Thanks for pointing it out Scott. They do get promoted and demoted if the filter changes. Admins do not maintain their status if the filter changes. Is this correct behaviour? Or should active admins maintain their status if the filter changes/removed?

Reply:
Yes, the admins should be promoted and demoted if the filter changes. The implementation is correct, just need to fix the text.

Reply:
Updated this text as provided by Dennis below!